en/case-studies/vanilla-survey.rst

```diff
@@ -0,0 +1,76 @@
+.. |br| raw:: html
+
+  <br/>
+
+Vanilla Surveys using QField
+============================
+
+By Ahi Saipaia (1), Leody Vainikolo (1), and John Duncan (2 and 3) |br|
+1 Ministry of Agriculture, Forestry, Food, and Fisheries, Government of Tonga |br|
+2 UWA School of Agriculture and Environment, The University of Western Australia |br|
+3 Geography and Planning, The University of Western Australia
+|br|
```

Comment:
Do we need these br?

Reply:
I don't know how to do to go to the line. If you have another suggestion I am a taker :)

Reply:
Also have a look at https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#line-blocks